frontend/src/metabase/nav/components/AppBar/AppBarLarge.tsx

```diff
@@ -35,26 +36,31 @@ const AppBarLarge = ({
   isNavBarVisible,
   isSearchVisible,
   isNewButtonVisible,
+  isBreadcrumbsVisible,
   isProfileLinkVisible,
   isCollectionPathVisible,
   isQuestionLineageVisible,
   onToggleNavbar,
   onLogout,
 }: AppBarLargeProps): JSX.Element => {
+  isNavBarOpen = isNavBarOpen && isNavBarVisible;
```

Comment:
I don't think introducing a new name would be easier to understand here giving that we already have both
- isNavBarOpen (is the sidebar toggled on?)
- isNavBarVisible ([from this bit of code](https://github.com/metabase/metabase/blob/2ff7b99771cf611f33c804b98fb4a5605f454440/frontend/src/metabase/selectors/app.ts#L98-L107))

which are already similar.

And to be fair `isNavBarOpen` (the state alone) isn't that useful. Since if the state is open, but it's not visible then we might as well now show it or say it's not open. That's why I decide to override the parameter here.

Reply:
I find it confusing, too. WDYT about renaming `isNavBarVisible` to something else, e.g. `isNavBarAvailable` / `isNavBarEnabled` or something like that. Then you could add a variable like `const isNavBarVisible = isNavBarOpen && isNavBarAvailable;` and don't overwrite an existing one